### PR TITLE
[pt-br] Update tutorials/hello-minikube.md

### PR DESCRIPTION
**Pull request by kirederik**
Updates the "Hello, Minikube" tutorial to be in sync with the english version of that page.

fixes https://github.com/kubernetes/website/issues/38919

### CODE REVIEW

File: content/pt-br/docs/tutorials/hello-minikube.md [hunk 1]
```diff
@@ -144,7 +169,7 @@ Por padrão, um Pod só é acessível utilizando o seu endereço IP interno no c
 
 5. (**Apenas no ambiente do Katacoda**) Observe o número da porta com 5 dígitos exibido ao lado de `8080` na saída do serviço. Este número de porta é gerado aleatoriamente e pode ser diferente para você. Digite seu número na caixa de texto do número da porta e clique em **Display Port**. Usando o exemplo anterior, você digitaria `30369`.
 
-Isso abre uma janela do navegador, acessa o seu aplicativo e mostra o retorno da requisição.
+    Isso abre uma janela do navegador, acessa o seu aplicativo e mostra o retorno da requisição.
```
Comment (stormqueen1990) [inline content/pt-br/docs/tutorials/hello-minikube.md:172]: 💯

**Review by stormqueen1990 — COMMENTED** [review]
Hello there, @kirederik! 👋 

Thank you very much for your contribution! 🎉 

I found a few bits that need some attention. If you have any questions or concerns, or disagrees with any of these points, please feel free to drop a line. I am also available on the Kubernetes Slack channel `#kubernetes-docs-pt` should you need any help. 😃 

Once you have addressed the feedback, please feel free to request a new review from me by writing `/cc @stormqueen1990` in a comment.

Thanks again!

File: content/pt-br/docs/tutorials/hello-minikube.md [hunk 2]
```diff
@@ -49,7 +49,32 @@ Se você instalou o Minikube localmente, execute: `minikube start`.
     minikube dashboard
     ```
 
-3. Apenas no ambiente do Katacoda: Na parte superior do terminal, clique em **Preview Port 30000**.
+3. Apenas no ambiente do Katacoda: Na parte superior to painel do terminal, clique no sinal de mais (+), e selectione **Select port to view on Host 1**.
```
[the remark this answers is not in the document]
Comment (kirederik) [reply content/pt-br/docs/tutorials/hello-minikube.md]: fixed!

File: content/pt-br/docs/tutorials/hello-minikube.md [hunk 3]
```diff
@@ -49,7 +49,32 @@ Se você instalou o Minikube localmente, execute: `minikube start`.
...
+
+4. Apenas no ambiente do Katacoda: Digite `30000`, e clique em **Display
+   Port**.
+
+{{< note >}}
+O comando `dashboard` habilita o complemento (_addon_) de dashboard e abre o proxy no navegador padrão.
+Voce pode criar recursos no Kubernetes, como Deployment e Serviço, pela dashboard.
```
[the remark this answers is not in the document]
Comment (kirederik) [reply content/pt-br/docs/tutorials/hello-minikube.md]: fair point! i initially left it as Service but found it translated in a few other pages 🤔 i agree that on this sentence specifically it makes more sense to leave it as Service. Fixed!

**Comment by kirederik** [comment]
feedback addressed!

/cc @stormqueen1990

**Review by stormqueen1990 — COMMENTED** [review]
/lgtm

**Review by yagonobre — APPROVED** [review]
Thanks!
/lgtm
/approve

**Pull request merged**